pkgs/desktops/plasma-5/plasma-workspace/0001-startkde.patch

```diff
@@ -48,21 +48,6 @@ index e0f7004..8ac41fd 100644
  }
  
  QStringList allServices(const QLatin1String& prefix)
-@@ -184,14 +184,6 @@ void runEnvironmentScripts()
-         }
-     }
-     sourceFiles(scripts);
--
```

Comment:
It is important that `startkde` doesn't contaminate the `XDG_DATA_DIRS` variable. Why is this removed?